### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by denisoster** [linuxdeepin/deepin-system-monitor#14]
Please add a display of the CPU temperature and the number of fan speeds.

**Comment by Jkasnese** [comment]
Yes, this would be good. I come from Ubuntu, where I used indicator-cpufreq to control my CPU temperature in order to prevent it from overheating and shutting down. Couldn't open it on Deepin, so if it was possible to set a limit to CPU temperature via System Monitor that would be great
